gitbook-docs/market/user-guides/circuit-developer.md

```diff
@@ -1,51 +1,64 @@
+[zkLLVM](https://github.com/NilFoundation/zkllvm) circuit compiler.
+You can find the required dependencies and build instructions in the
+[zkLLVM's repository](https://github.com/NilFoundation/zkllvm).
 
-Anyone can generate circuits. They are serialised & published on the proof market. This allows the reuse of the circuits by all other proof requesters.&#x20;
+Anyone can generate circuits.
+They are serialized & published on Proof Market, allowing reuse of the circuits by
+all proof requesters.
 
-## Create a new circuit (using [zkLLVM](https://github.com/NilFoundation/zkllvm))
+## Create a new circuit using [zkLLVM](https://github.com/NilFoundation/zkllvm)
 
-```
+```bash
 make -C ${ZKLLVM_BUILD:-build} <circuit target name> -j$(nproc)
 ```
 
 ## Prepare a statement with a circuit description for Proof Market
 
-Circuits are stored as a _statement_ structure on Proof Market. Statement description example can be found in /example directory
+Circuits are stored as a _statement_ structure on Proof Market.
+Example of a statement description can be found in the `/example` directory.
 
-```
+```bash
 python3 scripts/prepare_statement.py -c <zkllvm output> \
--o <statement description file> \ 
--n <statement name> -t <circuit type>
+    -o <statement description file> \ 
+    -n <statement name> -t <circuit type>
 ```
 
-Provide the necessary information listed in the output statement file
+Make sure to provide the necessary information listed in the output statement file.
 
-## Publish to Proof Market
+## Publish a statement to Proof Market
 
 {% hint style="info" %}
-Ensure you have done the [Authentication](sign-up.md) setup before progressing
+Make sure you've performed the [authentication setup](sign-up.md) before continuing.
 {% endhint %}
 
+You can find an authorization file example in the `/example` directory.
 
+The statement can be pushed to Proof Market via the Python script `statement_tools.py`:
 
-This statement can now be pushed to the Proof market via the Python script
-
-```
-python3 scripts/statement_tools.py push --file <json file with statement description> 
+```bash
+python3 scripts/statement_tools.py push 
+    --file <json file with statement description> 
 ```
 
-Authorisation file examples can be found in `/example` directory.
+The script will return an object containing a _\_key_ field — a unique descriptor of the statement.
 
-You will be returned an object containing a _\_key_ filed -- unique descriptor of the statement
+## Retrieving statements
 
-## Published circuit id
+A list of all available statements can be obtained by this command:
 
-A list of all available statements can be obtained by
-
-```
+```bash
 python3 scripts/statement_tools.py get
 ```
 
-\
+To retrieve a selected statement/circuit definition and its metadata
+uploaded upon the statement's publishing:
+
+```bash
+python3 scripts/statement_tools.py get \
+    --key <key of the statement> \
+    -o <output file> 
+```
```

Comment:
this one is almost perfect! There's one little detail to make the example easier to understand:

```suggestion
```bash
python3 scripts/statement_tools.py get \
    --key <key of the statement> \
    --output <output file> 
```
```

Reply:
Fixed